docs/semgrep-appsec-platform/slack.md

```diff
@@ -42,17 +42,18 @@ To install the Semgrep Slack app, follow these steps:
 To set up or subscribe to notifications for findings in your Slack workspace, perform the following steps:
 
 1. In your Slack workspace, find or create a channel for Semgrep notifications.
-2. In the selected Slack channel, enter the following slash command: `/semgrep_subscribe`. For private channels, first invite the Semgrep app by typing `@Semgrep` in the channel, then use the `/semgrep_subscribe` command to start receiving notifications.
-3. Optional: Enter the name of a specific project after `/semgrep_subscribe` to receive findings for that specific project only, for example, `/semgrep_subscribe acme-corp/vulnerable-repo`. The project must be entered in the following format:
+  * If you use a private channel for notifications, first invite the Semgrep app by typing `@Semgrep` in the channel. If the app is not invited to a private channel it cannot send notifications there.
+3. In the selected Slack channel, enter the following slash command: `/semgrep_subscribe`. 
+4. Optional: Enter the name of a specific project after `/semgrep_subscribe` to receive findings for that specific project only, for example, `/semgrep_subscribe acme-corp/vulnerable-repo`. The project must be entered as it is shown in Semgrep AppSec Platform, typically:
```

Comment:
```suggestion
4. Optional: Enter the name of a specific project after `/semgrep_subscribe` to receive findings for that specific project only. The project must be entered as it is shown in Semgrep AppSec Platform, typically:
```

Reply:
I'm curious why you're wanting to remove the example?

Reply:
oh got it, I didn't make the connection because the other one is a placeholder, but you're probably right they aren't both necessary!